apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -1059,3 +1085,8 @@ client_example() ->
         <<"recv_cnt">> => 4,
         <<"recv_msg.qos0">> => 0
     }.
+
+sessions_count(get, #{query_string := QString}) ->
+    Since = maps:get(<<"since">>, QString, undefined),
```

Comment:
The doc string and function spec of `keeper:count` suggest it should a `non_neg_number()`?
```suggestion
    Since = maps:get(<<"since">>, QString, 0),
```